src/main/webapp/app/quiz-statistics/quiz-point-statistic/quiz-point-statistic.component.ts

```diff
@@ -328,13 +330,21 @@ export class QuizPointStatisticComponent implements OnInit, OnDestroy, DataSetPr
             this.router.navigateByUrl('/quiz/' + this.quizExercise.id + '/quiz-statistic');
         } else {
             const previousQuestion = this.quizExercise.quizQuestions[this.quizExercise.quizQuestions.length - 1];
-            if (previousQuestion.type === QuizQuestionType.MULTIPLE_CHOICE) {
+            if (previousQuestion.type === this.MULTIPLE_CHOICE) {
```

Comment:
I have seen it a couple of times, that we assign enum properties as strings to components.
In my opinion this is not a good idea as this doesn't make it obvious that the constants are part of an enum.

I know you probably just wanted streamline it (which I think is good), but in this case I would rather keep it the old way or refactor it.

Note: You can assign enums as properties to components when you need to use them in a a template.
`protected QuizQuestionType = QuizQuestionType;`

Reply:
@thilo-behnke I have removed all readonly references of the enum property in the code except in the QuizStatisticsFooterComponent. When I apply your suggestion I get the following error and the footer is not displayed correctly anymore
> 'const' enums can only be used in property or index access expressions or the right hand side of an import declaration or export assignment or type query.

Reply:
Very strange, I didn't see that some of the `enums` are actually `const enums` (https://www.typescriptlang.org/docs/handbook/enums.html#const-enums).
In our case I don't see why are not using regular enums :thinking: 

If you create something new I would propose to use a normal enum and assign it, if the code was already this way I think it is better to keep it.